infrastructure/src/main/java/mops/filehandling/CsvReader.java

```diff
@@ -31,8 +32,13 @@ public CsvReader(MultipartFile file, Veranstaltung veranstaltung) {
     message = "";
     messageStatus = "";
 
-    if (isFileVerified()) {
-      readFromFile();
+    if (veranstaltung == null) {
+      setMessageAndStatus("Oh nein! Die Veranstaltung wurde nicht mitgegeben! "
+              + "<i>(Parameter null in CsvReader.java)</i>", messageError);
+    } else {
+      if (isFileVerified()) {
```

Comment:
Try to write this a little bit better. For example like this
```suggestion
if (veranstaltung != null && isFileVerified()) {
    readFromFile();
} else {
    setMessage....
}
```